charts/k0s/templates/_migrate.tpl

```diff
@@ -0,0 +1,6 @@
+{{/*
+  handles both replicas and syncer.replicas
+*/}}
+{{- define "vcluster.replicas" -}}
+{{ if .Values.syncer.replicas }}{{ .Values.syncer.replicas }}{{ else }}{{ .Values.replicas }}{{ end }}
```

Comment:
Shouldn't the if check .Values.replicas here as we set .Values.syncer.replicas now by default?